backend/tests/services/rag/test_remote_gateway.py

```
@@ -661,29 +651,155 @@ async def test_remote_gateway_list_chunks_posts_runtime_request(mocker) -> None:
+                ),
+                retrieval_config=RuntimeRetrievalConfig(),
+            )
+        ],
     )
 
-    assert isinstance(get_index_gateway(), LocalRagGateway)
-    assert isinstance(get_query_gateway(), RemoteRagGateway)
-    assert isinstance(get_delete_gateway(), RemoteRagGateway)
+    await gateway.query(spec)
+
+    args, kwargs = post_mock.await_args
```

Ruff (0.15.10)

[warning] Unpacked variable `args` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)